tests/switchless/enc/enc.c

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/string.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/print.h>
+#include "switchless_t.h"
+
+char* oe_host_strdup(const char* str)
+{
+    size_t n = oe_strlen(str);
+
+    char* dup = (char*)oe_host_calloc(1, n + 1);
```

Comment:
Since oe_shm_malloc alwaya returns null, dup is always null. Is that right?

Reply:
Hmmm no. See above.

Reply:
@jxyang thanks for the explanation